app/views/components/Button.scala.html

```diff
@@ -14,12 +14,21 @@
  * limitations under the License.
  *@
 
+@this(govukButton: GovukButton)
+
 @(
-messageKey: String = "",
-href: String,
-id: String
+msg: String = "button.saveandcontinue",
```

Comment:
Can you remove this default value for the `msg` parameter. All the views that call it provide their own button text - which is good.

Reply:
This is still not done